docs/artifactory/replicationtemplate/help.go

```diff
@@ -0,0 +1,8 @@
+package replicationtemplate
+
+const Description = "Create a JSON template for repository creation or update."
+
+var Usage = []string{`jfrog rt rtc <template path>`}
+
+const Arguments string = `	template path
+		Specifies the local file system path for the template file.`
```

Comment:
@liron-shalom is adding a suffix to this command, with the command that creates the template. Let's add the same here.

Reply:
Done